bitnami/orangehrm/requirements.lock

```diff
@@ -1,6 +0,0 @@
-dependencies:
```

Comment:
It looks like we are removing this file, is it a mistake?

Reply:
A _Chart.lock_ will be generated once `helm dep update` is executed